pkgs/development/libraries/tracker/default.nix

```diff
@@ -29,20 +29,21 @@
 
 stdenv.mkDerivation rec {
   pname = "tracker";
-  version = "3.0.3";
+  version = "3.1.0";
 
   outputs = [ "out" "dev" "devdoc" ];
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-b1yEqzvh7aUgUBsq7XIhYWoM8VKRDFN3V7U4vAXv/KM=";
+    sha256 = "sha256:0x98wk9347p9sf7vw0yhyswnzmd8hp01zczl5a0fyiw7fnaynmfa";
   };
 
   patches = [
     (substituteAll {
       src = ./fix-paths.patch;
       inherit asciidoc;
     })
+    ./fix-docs.patch
```

Comment:
Is there an upstream patch we can fetch instead?

Reply:
Not yet, I have created this patch on my own.

Reply:
Any progress in sending this upstream?

Reply:
@jtojnar Upstream will merge this patch for the 3.1.2 release, probably in the next two weeks-ish, see https://gitlab.gnome.org/GNOME/tracker/-/issues/292#note_1076661 .